README.md

```diff
@@ -211,9 +211,11 @@ Companies that have allowed us to speak about our work can be found here. Many m
 
 | Product | Review Date | Level of Effort | Deliverables | Announcement |
 | --- | --- | --- | --- | --- |
-| [VGS Proxy](https://www.verygoodsecurity.com/) | Apr 2021 | 4 person-weeks | |
-| [Skiff](https://www.skiff.org/) | Feb 2021 | 4 person-weeks | |
-| [CircleCI Server 3.0](https://circleci.com/) | Jan 2021 | 6 person-weeks | |
+| [DFINITY Internet Computer](https://dfinity.org/) | May 2021 | 24 person-weeks | | |
```

Comment:
This should be a blockchain project, and you can drop "Internet Computer" from the name

Reply:
updated pull request to reflect this change